omf-python/src/attribute.rs

```diff
@@ -93,6 +93,18 @@ impl PyAttributeDataCategory {
         }
     }
 
+    #[getter]
+    fn gradient(&self) -> PyResult<Option<PyGradientArray>> {
```

Comment:
We should start adding docstrings. We can probably often just take them directly from the rust documentation https://gmggroup.github.io/omf-rust/rust/omf/enum.AttributeData.html#variant.Category.field.gradient